orchestra/proc-macro/src/parse/parse_orchestra_struct.rs

```diff
@@ -56,8 +59,10 @@ impl Parse for SubSysAttrItem {
 			Self::Wip(input.parse::<kw::wip>()?)
 		} else if lookahead.peek(kw::blocking) {
 			Self::Blocking(input.parse::<kw::blocking>()?)
-		} else if lookahead.peek(kw::sends) {
+		} else if lookahead.peek(kw::consumes) {
```

Comment:
Was this intentional?

Reply:
Yes, it is an occasional IDE damage, but this is a draft PR, so it is not yet completed :)